### ISSUE TITLE
Default reference genome index file is a file in a 003 project

### ISSUE THREAD

**Issue by growland2** [eastgenomics/eggd_cgppindel#4]
Looking at all the projects where this file is present via dx find data, this file is not present in 001 Ref (project-Fkb6Gkj433GVVvj73J7x8KbV) (see code below)

```
$ dx describe --name file-G00jqQj4J7Byv963KkZ40kXv
GRCh38.no_alt_analysis_set_chr_mask21.fa.fai
$ dx find data --name GRCh38.no_alt_analysis_set_chr_mask21.fa.fai --all-project | grep file-G00jqQj4J7Byv963KkZ40kXv
closed  2022-03-10 11:34:38 8.00 KB   /test_cgppindel/GRCh38.no_alt_analysis_set_chr_mask21.fa.fai (project-FykvvJj4qyvb74pQFG3J21qZ:file-G00jqQj4J7Byv963KkZ40kXv)
closed  2022-02-15 09:28:51 8.00 KB   /reference/GRCh38.no_alt_analysis_set_chr_mask21.fa.fai (project-Fzp7bK04J7BvBjJb906zYxK0:file-G00jqQj4J7Byv963KkZ40kXv)
closed  2021-04-23 13:29:13 8.00 KB   /workflow/apps/GRCh38.no_alt_analysis_set_chr_mask21.fa.fai (project-G21BGP84q5JFYf168QjZ58Vz:file-G00jqQj4J7Byv963KkZ40kXv)
```

An identical copy of this file exists in 001 Ref under a different file-id (ile-FyFGvzQ41zgBq7xf4Gy9Q28g) which could be used (see code below). 
```
$ diff -s <(dx cat file-FyFGvzQ41zgBq7xf4Gy9Q28g) <(dx cat file-G00jqQj4J7Byv963KkZ40kXv)
Files /dev/fd/63 and /dev/fd/62 are identical
```

**Comment by jethror1** [comment]
@Arun-bioinformatics was this fixed in #5 and can this now be closed?

**Comment by Arun-bioinformatics** [comment]
Yes, this issue is now fixed:
https://github.com/eastgenomics/eggd_cgppindel/blob/main/dxapp.json#L53
